lib/node_modules/@stdlib/blas/ext/base/dsum/examples/index.js

```diff
@@ -18,11 +18,12 @@
 
 'use strict';
 
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
-var filledarrayBy = require( '@stdlib/array/filled-by' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
 var dsum = require( './../lib' );
 
-var x = filledarrayBy( 10, 'float64', discreteUniform( -100, 100 ) );
+var x = discreteUniform( 10.0, -100, 100, {
```

Comment:
```suggestion
var x = discreteUniform( 10, -100, 100, {
```

The first argument is a length, and thus always an integer value. We communicate this by omitting the decimal.